docker/Dockerfile.worker

```diff
@@ -0,0 +1,9 @@
+ARG ARTHUR_VERSION
+
+FROM grimoirelab/arthur:$ARTHUR_VERSION
+
+ENTRYPOINT [\
+  "/sbin/tini","--",\
+  "/bin/sh", "-c", \
+  "exec /usr/local/bin/python3 bin/arthurw -d ${ARTHUR_DATABASE} ${ARTHUR_WORKER_QUEUE}"\
+  ]
```

Comment:
Great to separate arthurd and the workers.  I am totally sure that we should have one worker per container, or start several workers in the same container. Looking to the future, the right approach is the one you are proposing now @olblak: one microservice (worker), one container. So the workers can be managed independently.